website/docs/troubleshooting/postgres/upgrade_kubernetes.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Upgrading PostgreSQL on Kubernetes
+---
+
+## Placeholders
```

Comment:
@BeryJu what is this section for (what does Placeholders mean?)?

Reply:
Basically just strings that are used in the comments below that might need to be replaced depending on the installation